test/test.js

```diff
@@ -1,24 +1,37 @@
 const got = require('got');
 const { expect } = require('chai');
-const server = require('../');
+const nock = require('nock');
 const { default: defaultValues } = require('../lib/server/defaultValues');
 
 describe('return type check', () => {
-    const newInstance = new server.MyServer();
+    it('Check 200 http and Name', async () => {
```

Comment:
Please revert the changes in this test. You can add a new test for the `/api/questions`.